src/main/java/com/example/Ingress_lab/service/UserService.java

```diff
@@ -0,0 +1,22 @@
+package com.example.Ingress_lab.service;
+
+import com.example.Ingress_lab.model.UserEntity;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Service;
+
+@Service
+@FieldDefaults( makeFinal = true, level = AccessLevel.PRIVATE)
+@RequiredArgsConstructor
+public class UserService {
+
```

Comment:
[Optional]: Added UserRepository and inject there, call findByUserId method and return mock data